sdk/documenttranslation/Azure.AI.DocumentTranslation/tests/SessionRecords/TranslationOperationLiveTests/TranslationOperationTestAsync.json

```diff
@@ -0,0 +1,313 @@
+        "x-ms-return-client-request-id": "true",
+        "x-ms-version": "2020-04-08"
+      },
+      "RequestBody": null,
+      "StatusCode": 201,
+      "ResponseHeaders": {
+        "Content-Length": "0",
+        "Date": "Wed, 24 Mar 2021 23:33:46 GMT",
+        "ETag": "\u00220x8D8EF1D455E7C38\u0022",
+        "Last-Modified": "Wed, 24 Mar 2021 23:33:46 GMT",
+        "Server": "Windows-Azure-Blob/1.0 Microsoft-HTTPAPI/2.0",
+        "x-ms-client-request-id": "e87882dcffcc8f503b73774710b64801",
+        "x-ms-request-id": "4f360b10-a01e-0075-2a06-21e813000000",
+        "x-ms-version": "2020-04-08"
+      },
+      "ResponseBody": []
+    },
+    {
+      "RequestUri": "https://nourdocuments.blob.core.windows.net/source601205785/document0.txt",
+      "RequestMethod": "PUT",
+      "RequestHeaders": {
+        "Authorization": "Sanitized",
+        "Content-Length": "39",
+        "If-None-Match": "*",
+        "traceparent": "00-57c7658b081d3b40bd86a20cebb6ab67-eb59edf5b0268f48-00",
+        "User-Agent": "azsdk-net-Storage.Blobs/12.8.0 (.NET Framework 4.8.4300.0; Microsoft Windows 10.0.19042 )",
+        "x-ms-blob-type": "BlockBlob",
+        "x-ms-client-request-id": "ab545b8d7b9fd496a009928e8e7e9419",
+        "x-ms-date": "Wed, 24 Mar 2021 23:34:13 GMT",
+        "x-ms-return-client-request-id": "true",
+        "x-ms-version": "2020-04-08"
+      },
+      "RequestBody": "VGhpcyBpcyB0aGUgZmlyc3QgZW5nbGlzaCB0ZXN0IGRvY3VtZW50",
+      "StatusCode": 201,
+      "ResponseHeaders": {
+        "Content-Length": "0",
+        "Content-MD5": "LO9siRJKhTnYlgGS/Ff6Qw==",
+        "Date": "Wed, 24 Mar 2021 23:33:47 GMT",
+        "ETag": "\u00220x8D8EF1D4638AF20\u0022",
+        "Last-Modified": "Wed, 24 Mar 2021 23:33:48 GMT",
+        "Server": "Windows-Azure-Blob/1.0 Microsoft-HTTPAPI/2.0",
+        "x-ms-client-request-id": "ab545b8d7b9fd496a009928e8e7e9419",
+        "x-ms-content-crc64": "3ufktO8PyY8=",
+        "x-ms-request-id": "4f360c37-a01e-0075-2806-21e813000000",
+        "x-ms-request-server-encrypted": "true",
+        "x-ms-version": "2020-04-08"
+      },
+      "ResponseBody": []
+    },
+    {
+      "RequestUri": "https://nourdocuments.blob.core.windows.net/source601205785/document1.txt",
+      "RequestMethod": "PUT",
+      "RequestHeaders": {
+        "Authorization": "Sanitized",
+        "Content-Length": "40",
+        "If-None-Match": "*",
+        "traceparent": "00-8f39da33631c9b40a50d6f183c9905fb-c14d7e15d39a0a48-00",
+        "User-Agent": "azsdk-net-Storage.Blobs/12.8.0 (.NET Framework 4.8.4300.0; Microsoft Windows 10.0.19042 )",
+        "x-ms-blob-type": "BlockBlob",
+        "x-ms-client-request-id": "0acaf3d3217e0399bb1d8e42fb468c99",
+        "x-ms-date": "Wed, 24 Mar 2021 23:34:13 GMT",
+        "x-ms-return-client-request-id": "true",
+        "x-ms-version": "2020-04-08"
+      },
+      "RequestBody": "VGhpcyBpcyB0aGUgc2Vjb25kIGVuZ2xpc2ggdGVzdCBkb2N1bWVudA==",
```

Comment:
is this a secret?

Reply:
@maririos It looks like a key yes. I don't know what it's for though. It's not the storage account key I'm using.

Reply:
hmmm looking again, this is in the `RequestBody` so this is the information we are sending about the blob to upload. def not a secret, so looks live we are good :)